EC-84: Revise Configuration, Customers, eContent, Login and Logging (#5)
ezsystems/commerce-documentation: docs/guide/econtent/econtent_features/variants_in_econtent.md

# Variants in eContent

`ses_variants` can be used to store variant data for a product. 

> The variants have to be stored in a JSON format.

Reviewer: ```suggestion
The variants must be stored in a JSON format.
```
